libraries/ui-library/src/components/six-sidebar-item/six-sidebar-item.tsx

```diff
@@ -22,20 +22,25 @@ export class SixSidebarItem {
   /** Set to true to draw the sidebar item in a disabled state. */
   @Prop({ reflect: true }) disabled = false;
 
+  /** Provide if the item should be rendered as anchor tag. */
+  @Prop({ reflect: true }) href: string | undefined;
+
   render() {
+    const Tag = this.href != null ? 'a' : 'div';
     return (
-      <div
+      <Tag
```

Comment:
I think we should allow links for `six-sidebar-item-group` too.

Reply:
done